README.md

```diff
@@ -16,24 +16,25 @@ More in depth documentation can be found in [The Act Book](https://ethereum.gith
 
 # Building
 
-With nix:
+You can build the project with nix. If you do not have nix installed yet, you can try using the [Determinate Nix installer](https://github.com/DeterminateSystems/nix-installer).
+
+Building with nix:
 
 ```sh
 nix build
 ```
 
 # Developing
 
-Enter a nix-shell to get the dependencies of the project:
+After building, enter a nix-shell to get the dependencies of the project:
 
 ```sh
 nix develop
 ```
 
-you can then use `cabal` as normal:
+you can then use `cabal` as normal from where you have `act.cabal`, could be in `./src`:
```

Comment:
`act.cabal` is at the root directory of act and it won't move (unless we move it).